aiotruenas_client/websockets/protocol.py

```diff
@@ -1,3 +1,4 @@
+from abc import abstractmethod
```

Comment:
Can you put this down with the other `from` imports?  I'm actually surprised `black` doesn't have an opinion on this.

Reply:
`black` doesn't handle import order, only the formatting.  You can add `isort` which makes this trivial for the developer.  I will move this.

Reply:
I opened #43 to do that in the future.